diff --git a/app/Http/Livewire/Admin/GradeController.php b/app/Http/Livewire/Admin/GradeController.php
index a8c9af95ea696f743771a2e68d27f75f0c1a6004..245a6fe274ad69a1e86cdab8e7f89f1334c6e38e 100644
--- a/app/Http/Livewire/Admin/GradeController.php
+++ b/app/Http/Livewire/Admin/GradeController.php
@@ -3,9 +3,9 @@
 namespace App\Http\Livewire\Admin;
 
 use App\Models\Coursesdetail;
+use App\Models\Group;
 use App\Models\Groupassignment;
 use App\Models\Inscription;
-use App\Models\Group;
 use App\Models\User;
 use Illuminate\Support\Facades\DB;
 use Livewire\Component;
diff --git a/app/Http/Livewire/Admin/GroupController.php b/app/Http/Livewire/Admin/GroupController.php
index 1e2fc8fa057b06e49050233a47ea831c9f5b2dfb..f4dec390f93b3293f6f8bbdd01f2e9db5b91dda7 100644
--- a/app/Http/Livewire/Admin/GroupController.php
+++ b/app/Http/Livewire/Admin/GroupController.php
@@ -135,7 +135,6 @@ public function delete()
 
     public function render()
     {
-
         return view('livewire.admin.groups.index', [
             'groups' => Group::where('nombre', 'like', '%'.$this->search.'%')
                      ->orWhere('capacidad', 'like', '%'.$this->search.'%')
