diff --git a/app/Http/Livewire/Admin/AreaController.php b/app/Http/Livewire/Admin/AreaController.php
index ed899d0d32da2639129c355da40033fa12e8ce0b..8cbf536339fcb719fe26605b862804061a057f86 100644
--- a/app/Http/Livewire/Admin/AreaController.php
+++ b/app/Http/Livewire/Admin/AreaController.php
@@ -44,7 +44,7 @@ public function resetFilters()
 
     private function validateInputs()
     {
-        if($this->edit==true){
+        if ($this->edit == true) {
             $this->validate([
                 'nombre' => ['required', 'regex:/^[\pL\pM\s]+$/u'],
                 'jefe_area' => ['required', 'regex:/^[\pL\pM\s]+$/u'],
@@ -53,12 +53,12 @@ private function validateInputs()
                 'telefono' => ['required', 'numeric'],
             ]);
         }
-        if($this->create==true){
+        if ($this->create == true) {
             $this->validate([
-                'nombre' => ['required', 'regex:/^[\pL\pM\s]+$/u','unique:areas'],
+                'nombre' => ['required', 'regex:/^[\pL\pM\s]+$/u', 'unique:areas'],
                 'jefe_area' => ['required', 'regex:/^[\pL\pM\s]+$/u'],
                 'extension' => ['required', 'numeric'],
-                'clave' => ['required', 'alpha_num','unique:areas'],
+                'clave' => ['required', 'alpha_num', 'unique:areas'],
                 'telefono' => ['required', 'numeric'],
             ]);
         }
diff --git a/app/Http/Livewire/Admin/GradeController.php b/app/Http/Livewire/Admin/GradeController.php
index 28021997fd505e62531e729197a1b9cb028c0e95..c0da9b0272cdcd8c766bd04c1741ab8a7c40bfba 100644
--- a/app/Http/Livewire/Admin/GradeController.php
+++ b/app/Http/Livewire/Admin/GradeController.php
@@ -80,7 +80,7 @@ public function edit($id)
                 ->select('users.id', 'course_details.id as course_details_id', DB::raw("concat(users.name,' ',users.apellido_paterno,' ', users.apellido_materno)as nombre"), 'courses.nombre as curso', 'groups.nombre as grupo', 'inscriptions.calificacion')
                 ->first();
         $this->grade_id = $id;
-        $this->course_details_id =$grade->course_details_id;
+        $this->course_details_id = $grade->course_details_id;
         $this->participante = $grade->nombre;
         $this->curso = $grade->curso;
         $this->grupo = $grade->grupo;
diff --git a/app/Http/Livewire/Admin/GroupController.php b/app/Http/Livewire/Admin/GroupController.php
index 56d2b24d6b68bb3e4e0d7679f8f883aa4f442cd5..975dd3d556721af7ff1d29c0299eb85aa8a19397 100644
--- a/app/Http/Livewire/Admin/GroupController.php
+++ b/app/Http/Livewire/Admin/GroupController.php
@@ -34,17 +34,16 @@ class GroupController extends Component
 
     private function validateInputs()
     {
-        if($this->create==true){
+        if ($this->create == true) {
             $this->validate([
-                'nombre' => ['required', 'alpha_num','unique:groups'],
+                'nombre' => ['required', 'alpha_num', 'unique:groups'],
             ]);
         }
-        if($this->edit==true){
+        if ($this->edit == true) {
             $this->validate([
                 'nombre' => ['required', 'alpha_num'],
             ]);
         }
-
     }
 
     public function updatingSearch()
diff --git a/app/Http/Livewire/Admin/PeriodCoursesController.php b/app/Http/Livewire/Admin/PeriodCoursesController.php
index 9bc6dfca0c332afbd70df699ea3c9ea8c27498aa..3358b4951610af097e5e7f98cb40ab10e621d895 100644
--- a/app/Http/Livewire/Admin/PeriodCoursesController.php
+++ b/app/Http/Livewire/Admin/PeriodCoursesController.php
@@ -29,13 +29,13 @@ public function resetFilters()
 
     public function render()
     {
-        return view('livewire.admin.periodCourses.index',[
+        return view('livewire.admin.periodCourses.index', [
             'periods'=>CourseDetail::distinct()
                      /* ->join('period_details','period_details.course_detail_id','course_detail_id') */
-                     ->join('periods','periods.id','course_details.period_id')
-                     ->join('courses','courses.id','course_details.course_id')
-                     ->join('groups','groups.id','course_details.group_id')
-                     ->select('courses.nombre as curso','groups.nombre as grupo','periods.fecha_inicio','periods.fecha_fin')
+                     ->join('periods', 'periods.id', 'course_details.period_id')
+                     ->join('courses', 'courses.id', 'course_details.course_id')
+                     ->join('groups', 'groups.id', 'course_details.group_id')
+                     ->select('courses.nombre as curso', 'groups.nombre as grupo', 'periods.fecha_inicio', 'periods.fecha_fin')
                      ->orderBy('periods.fecha_inicio')
                      ->paginate($this->perPage),
         ]);
